index.html

```diff
@@ -49,12 +49,39 @@
 
 <body>
     <header>
+        <div class="scroll-show">
+            <div class="white-bg"></div>
+            <div class="site-logo">
+                <a href="https://nigerialogos.com/"><img src="images/nigerialogos_logo.svg" alt="NigeriaLogos logo"></a>
+            </div>
+            <div class="secondary-alphabet">
+                <a href="##">#</a>
```

Comment:
Is there a significance for the double pound?

Reply:
@fathermerry Lol, I'm guessing because scroll to ID is `#`, so it's saying scroll to the div with id=#.

Reply:
@fathermerry exactly what @Odusanya said

Reply:
@demilad This is dope!

Reply:
thanks man! @aeadedoyin